scm-core/src/main/java/sonia/scm/repository/work/SimpleWorkingCopyFactory.java

```diff
@@ -100,11 +103,18 @@
   private static final Logger LOG = LoggerFactory.getLogger(SimpleWorkingCopyFactory.class);
 
   private final WorkingCopyPool workingCopyPool;
+  private MeterRegistry meterRegistry;
 
+  @Deprecated
   public SimpleWorkingCopyFactory(WorkingCopyPool workingCopyPool) {
     this.workingCopyPool = workingCopyPool;
   }
 
+  protected SimpleWorkingCopyFactory(WorkingCopyPool workingCopyPool, MeterRegistry meterRegistry) {
```

Comment:
Before this change `SimpleWorkingCopyFactory` could be instantiated without subclassing it. Is thins not longer allowed?